packages/agents/sdk/src/sdkUtils.ts

```diff
@@ -50,6 +51,16 @@ export class NxtpSdkUtils {
     return new NxtpSdkUtils(nxtpConfig, logger, chainData);
   }
 
+  public parseConnextTransactionReceipt(transactionReceipt: providers.TransactionReceipt): any {
+    const parsedlogs: any = [];
+    transactionReceipt.logs.forEach((log) => {
+      const l = this.contracts.connext.parseLog(log);
```

Comment:
This is gonna throw an error. We should make this function more specific and make the function:
```ts
function getTransferIdFromXCallReceipt(...)
```
Then we know exactly which log in the receipt to parse for.

Reply:
we can parse any log related to connext interface though, it doesn't have to be xcall right?